### PR TITLE
Have added all pools for penpie

### PR DESCRIPTION
**Pull request by jan-magpie**

### CODE REVIEW

File: constants/integration_ids.py [hunk 1]
```diff
@@ -30,6 +30,18 @@ class IntegrationID(Enum):
     STAKEDAO_SUSDE_JULY_LPT = ('stakedao_susde_july_effective_lpt_held', 'Stake DAO sUSDe July LPT', Token.SUSDE)
     STAKEDAO_SUSDE_SEPT_LPT = ('stakedao_susde_sept_effective_lpt_held', 'Stake DAO sUSDe Sept LPT', Token.SUSDE)
     STAKEDAO_USDE_JULY_LPT = ('stakedao_usde_july_effective_lpt_held', 'Stake DAO USDe July LPT', Token.USDE)
+    
+    #PENPIE 
+    PENPIE_SUSDE_JULY_LPT=('penpie_susde_july_effective_lpt_held', 'PENPIE sUSDe July LPT', Token.SUSDE)
+    PENPIE_Karak_sUSDe_26SEP2024_LPT= ('PENPIE_Karak_sUSDe_26SEP2024_effective_lpt_held', 'Karak_sUSDe_26SEP2024')
```
Comment (FJ-Riveros) [inline constants/integration_ids.py:36]: We are missing the `Token.SUSDE` specification here:

```suggestion
    PENPIE_Karak_sUSDe_26SEP2024_LPT= ('PENPIE_Karak_sUSDe_26SEP2024_effective_lpt_held', 'Karak_sUSDe_26SEP2024', Token.SUSDE)
```

File: constants/penpie.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+USDe25JUL2024_PRT="0x3984E25AF3EbcfB4ca8925e9fA8855e17DC1D240"
+USDe25JUL2024_PRT_DEPLOYMENT_BLOCK=19564654
+
+
+Zircuit_USDe27JUN2024_PRT="0x15254c0CD69cD5c92D6A8D66De487370E146b17d"
+Zircuit_USDe27JUN2024_PRT_DEPLOYMENT_BLOCK=19588125
+
+
+# USDe04APR2024_PRT=""
+# USDe04APR2024_PRT_DEPLOYMENT_BLOCK=
```
Comment (FJ-Riveros) [inline constants/penpie.py:35]: Could you remove this unused lines please?

File: integrations/pendle_lpt_integration.py [hunk 3]
```diff
@@ -83,7 +83,7 @@ def get_participants(self) -> list:
         if self.participants is not None:
             return self.participants
 
-        logging.info(f"[{self.get_description()}] Getting participants...")
+        # logging.info(f"[{self.get_description()}] Getting participants...")
```
Comment (FJ-Riveros) [inline integrations/pendle_lpt_integration.py:86]: Could you uncomment this line please?

File: utils/penpie.py [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+        userShare = userpenpeiepoolBal * 100 / penpeiepoolTotalSupply
+
+        print(user, userShare * lpt_bal / 100)
+        return userShare * lockerSyBalance / 100
+
+    def get_participants(self) -> list:
+        if self.participants is not None:
+            return self.participants
+
+        # logging.info(f"[{self.get_description()}] Getting participants...")
```
Comment (FJ-Riveros) [inline utils/penpie.py:145]: Could you uncomment this logs please?

File: utils/stakedao.py [hunk 5]
```diff
@@ -121,7 +121,7 @@ def get_participants(self) -> list:
         if self.participants is not None:
             return self.participants
 
-        logging.info(f"[{self.get_description()}] Getting participants...")
+        # logging.info(f"[{self.get_description()}] Getting participants...")
```
Comment (FJ-Riveros) [inline utils/stakedao.py:124]: Could you uncomment this line please?

**Review by FJ-Riveros — CHANGES_REQUESTED** [review]

File: constants/integration_ids.py (hunk 1 above)
Comment (jan-magpie) [reply constants/integration_ids.py:36]: is karak susde same as susde?

**Pull request merged**